Source/NSScrollView.m

```diff
@@ -1755,6 +1755,40 @@ - (id) initWithCoder: (NSCoder*)aDecoder
   return self;
 }
 
+- (BOOL)automaticallyAdjustsContentInsets
+{
+  // FIXME
+  return NO;
+}
+
+- (void)setAutomaticallyAdjustsContentInsets: (BOOL)adjusts
+{
+  // FIXME
+  return;
+}
+
+- (NSEdgeInsets)contentInsets
+{
+  // FIXME
+  return NSEdgeInsetsZero;
+}
+- (void)setContentInsets:(NSEdgeInsets)edgeInsets
+{
+  // FIXME
+  return;
+}
+
+- (NSEdgeInsets)scrollerInsets
+{
+  // FIXME
+  return NSEdgeInsetsZero;
+}
+- (void)setScrollerInsets:(NSEdgeInsets)insets
+{
+  // FIXME
+  return;
```

Comment:
Any reason for the empty return?

Same comment from the other review applies, we might want to print out warnings.

In addition to that, in these cases where we don't depend on `CALayer`'s existence to do something smart, it might make sense to at least store the passed values and return them correctly, even if they do nothing else?